app/src/main/res/layout/component_share.xml

```diff
@@ -10,18 +10,20 @@
     android:layout_height="match_parent"
     android:clipToPadding="false">
 
-    <Button
+    <com.google.android.material.button.MaterialButton
         android:id="@+id/close_button"
+        style="@style/Widget.MaterialComponents.Button.TextButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_margin="16dp"
         android:background="?android:attr/selectableItemBackgroundBorderless"
```

Comment:
I think `android:background="?android:attr/selectableItemBackgroundBorderless"` is redundant now.

Reply:
Nice catch!